### PR TITLE
Clean up / rewrite "Backing Up Data" wiki article

### PR DESCRIPTION
**Pull request by FloatingMilkshake**
Cleaned up the "Backing Up Data" wiki article. The gist of the article is the same, but I felt it could be made a little clearer.

(also set widths for images)

### CODE REVIEW

File: wiki/backup.md [hunk 1]
```diff
@@ -1,86 +1,98 @@
...
 ## The 3-2-1 Backup Rule
 
-The 3-2-1 rule is keeping 3 copies of your data, two on different storage mediums (your computer's drive and a backup drive for example) and one on the cloud.
+The "3-2-1" backup rule is a good guideline to follow when backing up your data. It helps ensure that you always have a copy of your data, even if something happens to a backup.
 
-In other words, if you have a file called "image.jpg" on your computer's main drive, you can create another copy on your external backup drive and/or on the cloud. Now you have 3 copies of image.jpg.
+The 3-2-1 rule is as follows:
+- Keep at least 3 copies of your data
+- Use at least 2 different storage media
+- Keep at least one copy off-site
```
Comment (waitwhatActo) [inline wiki/backup.md:16]: maybe use "1" instead of "one" for consistency?

Comment (FloatingMilkshake) [reply wiki/backup.md]: Good catch, thanks! Changed

**Review by Erisa — APPROVED** [review]

**Review by waitwhatActo — APPROVED** [review]

**Comment by FloatingMilkshake** [comment]
Thanks for the feedback, will merge
